src/actions/notifications.ts

```diff
@@ -256,11 +255,8 @@ export function configureNotificationHandler() {
         !__DEV__ && rollbar.error(error);
       },
 
-      // ANDROID ONLY: GCM Sender ID
-      senderID: GCM_SENDER_ID,
-
-      // we manually call this after to have access to a promise for the iOS prompt
-      requestPermissions: false,
+      // we manually call this after on iOS to have access to a promise for the iOS prompt
+      requestPermissions: isAndroid,
```

Comment:
I think this is what fixed things. Thanks for figuring that out!

Reply:
Yay, glad this worked 🙂

Reply:
Just wanted to document things. This fix was originally discussed in https://github.com/CruGlobal/missionhub-react-native/pull/2019.